### PR TITLE
Update fixture and local/test E2E assets for Bttajk services 

### LINKED ISSUE #4813: Update fixture and local/test E2E assets for 6 services
**Is your feature request related to a problem? Please describe.**
We need to run standard tests on six new services we want to launch.

**Describe the solution you'd like**
Add the following assets from Test to fixture data:

Burmese cn0exdy1jzvo
Thai c442rl3md0eo
Tigrinya ck62z3rjwdeo
Amharic czqverekrldo
Japanese cdd6p3r9g7jo
Korean c3mn1lvz65xo

And update the Cypress tests for these services to use these assets for testing on local and test envs. We need to change the config file for test.

These assets haven't been created yet, we need to add the assets from the Ares data. https://bbcnews.slack.com/archives/CHTF3GCTH/p1575025122016600 Follow slack message to access the data.

Run the Cypress tests against both envs to confirm they pass as expected.

**Describe alternatives you've considered**
N/A

**Testing notes**
Cypress smoke test need to be set to false for these new assets.

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [x] This feature is expected to need manual testing.

**Additional context**
Refer to commands at the base of the PR template when running these tests.

### PR DESCRIPTION
**Pull request by RayNjeri**
Signed-off-by: RayNjeri <rachael.njeri@andela.com>

Resolves #4813

**Overall change:** _Adds fixture data and live test assets for the 6 services._

**Code changes:**

- _Adds IDs for live test assets for Burmese, Thai, Tigrinya, Amharic, Japanese and Korean services._
- Pulls in fixture data from ares

---

- [x] I have assigned myself to this PR and the corresponding issues
- [ ] Automated (jest and/or cypress) tests added (for new features) or updated (for existing features)
- [x] If necessary, I have run the local E2E non-smoke tests relevant to my changes (`CYPRESS_APP_ENV=local CYPRESS_SMOKE=false npm run test:e2e:interactive`) 
- [x] This PR requires manual testing

### CODE REVIEW

**Review by HarveyPeachey — APPROVED** [review]
LGTM. Checked all assets on test and local

File: cypress/support/config/services.js [hunk 1]
```diff
@@ -84,7 +84,9 @@ const genServices = appEnv => ({
     variant: 'default',
     pageTypes: {
       articles: {
-        path: isLive(appEnv) ? undefined : '/amharic/articles/c3rykrrvy19o',
+        path: isLive(appEnv)
+          ? '/amharic/articles/czqverekrldo'
+          : '/amharic/articles/c3rykrrvy19o',
```
Comment (jimjohnsonrollings) [inline cypress/support/config/services.js]: These new assets are on Test/Local, which is the second line here.  The first line is for live assets, which we don't yet have for these services.

**Review by jimjohnsonrollings — CHANGES_REQUESTED** [review]

File: cypress/support/config/services.js (hunk 1 above)
Comment (RayNjeri) [reply cypress/support/config/services.js]: Addressed and updated.

Comment (jimjohnsonrollings) [reply cypress/support/config/services.js]: 👍

**Review by jimjohnsonrollings — APPROVED** [review]

**Comment by jimjohnsonrollings** [comment]
👍

**Review by jamesdonoh — APPROVED** [review]

**Comment by jamesbrumpton** [comment]
lgtm

**Review by HarveyPeachey — APPROVED** [review]
LGTM

**Pull request merged**